app/lib/lnd/methods/channelController.js

```diff
@@ -19,13 +19,14 @@ function ensurePeerConnected(lnd, pubkey, host) {
  * @return {[type]}         [description]
  */
 export function connectAndOpen(lnd, event, payload) {
-  const { pubkey, host, localamt } = payload
+  const { pubkey, host, localamt, private: privateChannel } = payload
```

Comment:
Why do we need to change the variable name to `privateChannel`? Can't it just remain named `private`? especially as that is what we want it to be named when we pass it into `lnd.openChannel` below.

Reply:
Because private is a reserved word in strict mode